resources/metadata/cloudFoundryDeploy.yaml

```diff
@@ -14,6 +14,11 @@ spec:
       - name: dockerCredentialsId
         description: Jenkins 'Username with password' credentials ID containing user and password to authenticate to the Docker registry.
         type: jenkins
+    resources:
+      - name: deployDescriptor
+        type: stash
+      - name: pipelineConfigAndTests
```

Comment:
not sure if this is really necessary ...

Reply:
You are right it's not required, since the `piperExecuteBin` step already unstashes it. Have removed it with the latest commit.